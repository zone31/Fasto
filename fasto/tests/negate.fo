fun bool main() =
    let x0 =  write(3 / 2 ==  1) in
    let x1 = write(~3 / 2 == ~2) in
    let x2 =  write(3 /~2 == ~2) in
    let x3 = write(~3 /~2 ==  1) in
    let x4 = write(~(5+5) ==  ~10) in
    let x5 = write(~(5+5)) in
    let x5 = write(~(1)) in
    write(x0 && x1 && x2 && x3 && x4)
