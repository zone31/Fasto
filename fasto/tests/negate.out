TrueFalseFalseTrueTrue-10 -1 False
